src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -94,7 +94,7 @@ export const thorchainApi: SwapperApi = {
         const feeData = await getFees({
           adapter: assertGetEvmChainAdapter(chainId),
           data,
-          to,
+          to: router,
```

Comment:
l1 swaps are sent to the router

Reply:
this one was actually accurate fortunately, but this update makes things more clear and explicit moving forward